build_library/release_util.sh

```diff
@@ -229,18 +229,23 @@ download_image_url() {
     local download_root="${FLAGS_download_root:-${UPLOAD_ROOT}}"
 
     local download_path
+    local download_channel
     if [[ -n "${FLAGS_download_path}" ]]; then
         download_path="${FLAGS_download_path%%/}"
-    elif [[ "${download_root}" = *release.core-os.net* ]]; then
+    elif [[ "${download_root}" = *flatcar-jenkins* ]]; then
         # Official release download paths don't include the boards directory
```

Comment:
would move this comment to after the `if`

Reply:
Makes sense. I've moved the comment after the `if`.